### DISCUSSION TITLE
Plotman interactive not showing active plots

### DISCUSSION THREAD

**Discussion by glasscity** [ericaltendorf/plotman#849] [General]
**Describe the bug**
When I start and run plotman interactive it does not show active current plots.

**System setup:**
 - OS: Debian 11 
 - Plotman installed with .deb

**Comment by altendky** [comment]
plotman isn't entirely and completely broken for everyone, at least not in the way you describe.  We'll need to see the evidence that makes you say that there are active plots that aren't being shown.  Also, there is no (official) plotman `.deb`.

**Reply by glasscity** [reply]
Yes, silly mistake on my part. Chia was installed with a .deb. This problem started after updating to 0.5
![Pscreen](https://user-images.githubusercontent.com/757525/125205716-a253da00-e2be-11eb-906f-a692f539276d.png)

**Reply by altendky** [reply]
Evidence that a job is running might come from `ps aux | grep 'plots create'`.  It would also be good to see the plotting log file for that job.

**Reply by glasscity** [reply]
Sorry about that. I did not think to add the log file. Hope this is whats needed. Thanks

![Pcapture](https://user-images.githubusercontent.com/757525/125274593-13d86a80-e349-11eb-9297-344828e959a6.png)
![Pgrep](https://user-images.githubusercontent.com/757525/125274613-1935b500-e349-11eb-8110-5b8281401da6.png)
[2021-07-12T14_56_41.219393+09_00.plot.log](https://github.com/ericaltendorf/plotman/files/6800700/2021-07-12T14_56_41.219393%2B09_00.plot.log)

**Reply by altendky** [reply]
I'm guessing you redacted the farmer public key and pool contract address?  Would be better to use a color other than the background so it is particularly explicit that you did.  (though they are public `:]`)

I haven't come up with any explanation yet so I'll have to dig into it in more detail later, sorry.  Thanks for the extra info.  Any chance that `plotman status` happens to find anything?  I don't expect so, but something to check just in case.

**Reply by glasscity** [reply]
Yes I redacted the key and address. In the future I'll keep in mind that they are public or redact with a color. Thanks for the advice. Also, plotman status has the same result and shows 0 jobs. Thanks again for looking into it.

**Reply by altendky** [reply]
I think I've got a fix over in https://github.com/ericaltendorf/plotman/pull/865.  This was also reported over in https://github.com/ericaltendorf/plotman/issues/862.  If you want to try it out you can use the regular installation instructions but with `altendky/plotman@fix_chia_binary_process_detection` instead of `ericaltendorf/plotman@main`.  (in case someone passes by later, this branch will be deleted once merged)

**Reply by glasscity** [reply]
I updated and everything is working as intended. Thanks again.

**Reply by altendky** [reply]
Thanks for testing.  I'll plan to get a small release out soon.

**Reply by ilovepancakes95** [reply]
@altendky I am experiencing the same issue, I know plot is running because I can see the files growing on my SSD, but plotman status shows nothing and interactive shows no plots running, and it keeps starting a new plot every 20 seconds because it thinks no plots are running even though they are. If I already have plotman installed from the main branch, how do I overwrite that install with your fixed branch to test? Or if you are putting a small release out today let me know and I'll just wait for that.

**Reply by altendky** [reply]
I'm not going to promise today, but maybe, we'll see.  If you follow the regular install instructions in the readme but substitute `altendky/plotman@fix_chia_binary_process_detection` instead of `ericaltendorf/plotman@main` in the URL, you should get the fix branch.

**Reply by ilovepancakes95** [reply]
> I'm not going to promise today, but maybe, we'll see. If you follow the regular install instructions in the readme but substitute `altendky/plotman@fix_chia_binary_process_detection` instead of `ericaltendorf/plotman@main` in the URL, you should get the fix branch.

No worries, and okay that will just overwrite the existing branch and not install 2 separate ones right?

**Reply by altendky** [reply]
It will.  To get back to `main` after a release you can just run the regular install instructions again without modification.

**Reply by ilovepancakes95** [reply]
> It will. To get back to `main` after a release you can just run the regular install instructions again without modification.

That branch on your repo couldn't be found by install command anymore so used your v0.5.1 branch and that seems to fix the issue. Thank you!

**Reply by hajes** [reply]
I had something similar...log went to non-existent folder and plotman couldn't find any running plot.
